src/allotropy/parsers/example_weyland_yutani/example_weyland_yutani_parser.py

```diff
@@ -0,0 +1,90 @@
+from io import IOBase
+import uuid
+
+from allotropy.allotrope.allotrope import AllotropyError
+from allotropy.allotrope.models.fluorescence_benchling_2023_09_fluorescence import (
+    ContainerType,
+    DeviceControlAggregateDocument,
+    DeviceControlDocumentItem,
+    DeviceSystemDocument,
+    MeasurementAggregateDocument,
+    MeasurementDocumentItem,
+    Model,
+)
+from allotropy.allotrope.models.shared.components.plate_reader import (
+    ProcessedDataAggregateDocument,
+    ProcessedDataDocumentItem,
+    SampleDocument,
+)
+from allotropy.allotrope.models.shared.definitions.custom import (
+    TQuantityValueNumber,
+)
+from allotropy.allotrope.models.shared.definitions.definitions import TDateTimeValue
+from allotropy.parsers.example_weyland_yutani.example_weyland_yutani_structure import (
+    Data,
+)
+from allotropy.parsers.lines_reader import CsvReader
+from allotropy.parsers.vendor_parser import VendorParser
+
+
+class ExampleWeylandYutaniParser(VendorParser):
+    def _parse(self, raw_contents: IOBase, _: str) -> Model:
+        reader = CsvReader(raw_contents)
+        return self._get_model(Data.create(reader))
+
+    def _get_model(self, data: Data) -> Model:
+        if data.number_of_wells is None:
+            msg = "Unable to get number of the wells in the plate"
+            raise AllotropyError(msg)
+
+        return Model(
+            measurement_aggregate_document=MeasurementAggregateDocument(
+                measurement_identifier=str(uuid.uuid4()),
+                measurement_time=self._get_measurement_time(data),
+                analytical_method_identifier=data.basic_assay_info.protocol_id,
+                experimental_data_identifier=data.basic_assay_info.assay_id,
+                container_type=ContainerType.well_plate,
+                plate_well_count=TQuantityValueNumber(value=data.number_of_wells),
+                device_system_document=DeviceSystemDocument(
+                    model_number=data.instrument.serial_number,
+                    device_identifier=data.instrument.nickname,
+                ),
+                measurement_document=self._get_measurement_document(data),
+            )
+        )
+
+    def _get_measurement_time(self, data: Data) -> TDateTimeValue:
+        return self.get_date_time(
+            data.instrument.serial_number or "2023-12-31"
+        )  # FIXME
```

Comment:
we should certainly add this to our README/guidelines, but could you change comments like this to start with `TODO` and a short description?

Reply:
fixed as requested